src/api/app/controllers/webui/request_controller.rb

```diff
@@ -15,7 +15,12 @@ def add_reviewer_dialog
   end
 
   def add_reviewer
-    required_parameters :review_type
+    unless params[:review_type]
```

Comment:
Why is `:review_type` not in the routes?

Reply:
Because `review_type` comes from a select_tag

Reply:
So?

Reply:
`required_parameters :review_type` line and `params[:review_type]` check dropped, by the following reasons:
- `:review_type` can be in the routes, but coming from a `select_tag` makes it not possible to include it in the url helper in the form.
- As `review_type` comes from fixed values in a `select_tag` we can rely on the code of the `rescue` clauses to deal with values not included in the `select_tag`.